lib/collections.js

```diff
@@ -47,7 +82,24 @@ longAndShort['longest'] = '';
 longAndShort['shortest'] = paragraph;
 
 /* your code starts here */
-
+var j = 0;
+for (var i = 0; i < uniqueWords.length; i++) {
+  if (uniqueWords[i].length > j) {
+    j = uniqueWords[i].length;
+    longAndShort['longest'] = uniqueWords[i];
+  }
+}
+
+var k = 5;
```

Comment:
Why `5`? Actually, this will cause your code to not work if none of the words are fewer than five letters long.